### DISCUSSION TITLE
Filter OR

### DISCUSSION THREAD

**Discussion by LucasLeandro1204** [perspective-dev/perspective#2501] [Q&A] [answered]
I remember that in a previous version we could pass the "filter_op" setting to switch between the "and" and "or" operations. Is it still possible, if not, why?

**Comment by texodus** [comment, marked as answer]
This is no longer necessary - you can use `expressions` to create arbitrary boolean filter conditions:
```javascript
viewer.restore({
    expressions: {
        "My Condition": "(\"Sales\" > 100 or \"Profit\" < 100) and \"Quantity\" > 1"
    },
    filter: [["My Condition", "==", true]]
})
```

**Reply by LucasLeandro1204** [reply]
Thank you!
